MANIFEST.in

```diff
@@ -1 +1,3 @@
 include algosdk/data/langspec.json
+global-include *.pyi
```

Comment:
@jdtzmn @algochoi Can you help me understand why `MANIFEST.in` is required?

* I'm asking the question having done minimal research.  From cursory review, my sense is the file is _not_ needed.  I scanned links like https://stackoverflow.com/questions/24727709/do-python-projects-need-a-manifest-in-and-what-should-be-in-it and https://www.remarkablyrestrained.com/python-setuptools-manifest-in/.
* Admittedly, I don't yet have a clear working understanding of `sdist`.  Though, the following `grep` suggests the desired files _are_ included even when I locally delete `MANIFEST.in`.
```
❯ python setup.py sdist | grep -E 'algosdk/py.typed|langspec.json'
/Users/michael/dev/py-algorand-sdk/.venv/lib/python3.8/site-packages/setuptools/dist.py:463: UserWarning: Normalizing 'v1.20.1' to '1.20.1'
  warnings.warn(tmpl.format(**locals()))
warning: check: missing required meta-data: url

copying algosdk/py.typed -> py-algorand-sdk-1.20.1/algosdk
copying algosdk/data/langspec.json -> py-algorand-sdk-1.20.1/algosdk/data
``` 
* Aside:  The above command shows some warnings suggesting `setup.py` is malformed.  If it's not a big lift, it feels worthwhile to amend `setup.py` + to understand how to prevent warnings.  Notably, I do _not_ see these errors in pyteal.

Reply:
* It sounds okay to me if we remove `MANIFEST.in` as well. 
* It sounds prudent to add the `url` field to our `setup.py`, as that[ metadata is now required ](https://docs.python.org/3/distutils/setupscript.html#additional-meta-data)by `setup()`.

Reply:
I have amended the `setup.py` file to include the `url` field.

**py-algorand-sdk requires `MANIFEST.in` because it support minimum Python version 3.8**

I tested the `sdist` behavior when removing the `MANIFEST.in` file and similarly found that the `algosdk/py.typed` file was being copied as expected. However, on Python v3.8.15 with `setuptools` v56.0.0, files demarcated in `setup.py`'s `package_data` are included only when building with `bdist` and not `sdist` ([source](https://stackoverflow.com/a/14159430)). Therefore, it is still necessary to include `MANIFEST.in`.

Note that this behavior is hidden until the build cache files are deleted. 

**PyTeal does not need `MANIFEST.in` because Python 3.10 implicitly copies files from `package_data`**

Further testing revealed that this is not an issue within PyTeal as Python v3.10 and up appears to correctly copy files from `package_data` without needing `MANIFEST.in`.